src/Parallel/Parallel.hpp

```diff
@@ -0,0 +1,157 @@
+// Distributed under the MIT License.
```

Comment:
I was thinking to split out the printf function (and its helpers) into a Parallel/Print.hpp  (or Printf.hpp) and have the remaining functions in Parallel/Info.hpp.  What do you think?